Makefile

```diff
@@ -240,20 +240,21 @@ ifneq ($(origin CATALOG_BASE_IMG), undefined)
 FROM_INDEX_OPT := --from-index $(CATALOG_BASE_IMG)
 endif
 
-# Build a catalog image by adding bundle images to an empty catalog using the operator package manager tool, 'opm'.
-# This recipe invokes 'opm' in 'semver' bundle add mode. For more information on add modes, see:
-# https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
+# Build an OLM catalog image by adding the bundle image to a simple catalog using the
+# operator package manager tool, 'opm'. For more information see:
+# https://olm.operatorframework.io/docs/reference/catalog-templates
 .PHONY: catalog-build
-catalog-build: opm ## Build a catalog image.
-	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+catalog-build: opm ## Build an OLM catalog image (for testing).
+	cat catalog/template.yaml | BUNDLE_IMG=$(BUNDLE_IMG) VERSION=$(VERSION) envsubst > catalog/_template.yaml
+	$(OPM) alpha render-template basic -o yaml < catalog/_template.yaml > catalog/catalog.yaml
```

Comment:
I get the following when running this:
```
make[1]: Leaving directory '/home/michele/Engineering/cloud-patterns/patterns-operator'
cat catalog/template.yaml | BUNDLE_IMG=quay.io/rhn_support_mbaldess/patterns-operator-bundle:v6.6.6 VERSION=6.6.6 envsubst > catalog/_template.yaml
/home/michele/Bin/opm alpha render-template basic -o yaml < catalog/_template.yaml > catalog/catalog.yaml
Error: unknown shorthand flag: 'o' in -o
```
I also tried a 4.12.x opm binary to no avail. Maybe something missing in there?

Reply:
Weird, it worked for me. I’ll check it out tonight.

FYI The docs for OLM indicate that -o yaml should be valid. https://olm.operatorframework.io/docs/tasks/creating-a-catalog/#generating-the-catalog